### PR TITLE
Adds "RPL" to notifications

### PR DESCRIPTION
**Pull request by Buttaa**
Adds “RPL” to Rocket Pool-related notifications to avoid confusion for non-RPL users.

### CODE REVIEW

File: frontend/locales/en.json [hunk 1]
```diff
@@ -768,7 +768,7 @@
       "event_type": {
         "gas_above": "Gas price maximum reached",
         "gas_below": "Gas price minimum reached",
-        "new_reward_round": "New reward round",
+        "new_reward_round": "New RPL reward round",
```
Comment (marcel-bitfly) [inline frontend/locales/en.json:771]: fyi: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/abbr#accessibility

> Spelling out the acronym or abbreviation in full the first time it is used on a page is beneficial for helping people understand it, especially if the content is technical or industry jargon.

**Comment by marcel-bitfly** [comment]
np: `commit message`

feat(notifications): adding `RPL` to `labels`

To make it easier to understand for user what the information belongs to.

See: BEDS-XXX

**Comment by benji-bitfly** [comment]
This `PR` will be closed, as the implementation can now be found here: https://github.com/gobitfly/beaconchain/pull/1092

**Comment by benji-bitfly** [comment]
This `PR` will be closed, as the implementation can now be found here: https://github.com/gobitfly/beaconchain/pull/1092

**Comment by marcel-bitfly** [comment]
thx for helping out though @Buttaa...keep up the spirit 😅

**Pull request closed without merging**